### PR TITLE
Update tensorflow cuda and vs code notebook images

### PR DESCRIPTION
**Pull request by kindomLee**
* Update tensorflow cuda signing key
* Update code server version and install ipykernel

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: components/example-notebook-servers/jupyter-tensorflow/cuda.Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
 ENV NVIDIA_REQUIRE_CUDA "cuda>=${CUDA_VERSION}"
 
 # install - other nvidia stuff
-RUN curl -sL "https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/7fa2af80.pub" | apt-key add - \
```
Comment (kindomLee) [inline components/example-notebook-servers/jupyter-tensorflow/cuda.Dockerfile:37]: I think they haven't changed their sign key in machine-learning repo

**Comment by kindomLee** [comment]
lol
Where can I save the code-server base image created in the previous step, so that I can use and put in the dockerfile in a later step

**Comment by DavidSpek** [comment]
The base image update will need to be merged first, then in a separate PR you can use it after it’s built.

**Comment by juliusvonkohout** [comment]
Can you join the next notebooks WG meeting in two weeks ? We are always looking for help.

https://www.kubeflow.org/docs/about/community/#kubeflow-community-calendars has the calendar and meeting notes.

You can also reach us on slack in the channel wg-notebooks and wg-manifests.

**Comment by juliusvonkohout** [comment]
/close

We have another PR to update all images

**Pull request closed without merging**